docs/admin/authorization/index.md

```diff
@@ -7,56 +7,43 @@ assignees:
 title: Overview
 ---
 
-In Kubernetes, authorization happens as a separate step from authentication.
-See the [Accessing Control Overview](/docs/admin/accessing-the-api/) for an
-overview of how authentication and authorization are applied to requests.
+* TOC
```

Comment:
The TOC is automatically added in the template, so you can remove lines 10 and 11, i.e.:

* TOC
{:toc}